services/user/CommonApi/common/packages/common-lib/src/supervisor.ts

```diff
@@ -121,23 +122,31 @@ export class Supervisor {
         const expected = this.pendingRequest.call;
         const received = response.call;
         const unexpected =
-            expected.method != received.method ||
-            expected.service != received.service;
+            expected.method !== received.method ||
+            expected.service !== received.service;
+
+        const {result} = response;
```

Comment:
Is your prettier running? Because this suggests it's not and the next dev who visits is gonna end up making formatting changes. 
```suggestion
        const { result } = response;
```

Reply:
Fixed. I ran prettier in the `Supervisor` project but forgot `common-lib`.
Also, I've spent some time this morning figuring out how to set up [multi-root workspaces](https://code.visualstudio.com/docs/editor/multi-root-workspaces) in vscode... it's pretty cool. And now prettier should always be running on save by default.